catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -123,6 +159,27 @@ public void testInPlaceMonitoring()
     }
 
     @Test
+    public void testInPlaceDavMonitoring()
+            throws IOException, InterruptedException, SourceUnavailableException, IngestException,
```

Comment:
I dislike that pattern, especially since the IDE manages them for you, but I remain open to having my mind changed.